cli/js/permissions.ts

```diff
@@ -47,17 +47,17 @@ export class PermissionStatus {
 }
 
 export class Permissions {
-  async query(desc: PermissionDescriptor): Promise<PermissionStatus> {
+  query(desc: PermissionDescriptor): PermissionStatus {
     const state = permissionsOps.query(desc);
     return new PermissionStatus(state);
   }
 
-  async revoke(desc: PermissionDescriptor): Promise<PermissionStatus> {
+  revoke(desc: PermissionDescriptor): PermissionStatus {
     const state = permissionsOps.revoke(desc);
     return new PermissionStatus(state);
   }
 
-  async request(desc: PermissionDescriptor): Promise<PermissionStatus> {
+  request(desc: PermissionDescriptor): PermissionStatus {
     const state = permissionsOps.request(desc);
     return new PermissionStatus(state);
```

Comment:
Wrap the return types here in `Promise.resolve()`.

Reply:
Gotcha. Thanks!